pysc2/lib/renderer_human.py

```diff
@@ -617,6 +630,27 @@ def draw_units(self, surf):
     """Draw the units and buildings."""
     for u, p in self._visible_units():
       if self._camera.intersects_circle(p, u.radius):
+        screen_loc = surf.world_to_surf.fwd_pt(p).floor()
+        aligned_screen_loc = point.Point(screen_loc.x - (self.scale * u.radius * 1.3),
+                                         screen_loc.y - (self.scale * u.radius * 1.35))
+        if u.shield:
+          surf.draw_arc(colors.blue,
+                        aligned_screen_loc,
+                        u.radius,
```

Comment:
Using u.radius both here and below means they'll be drawn as the same arc, right? It'd probably be better to send slightly different draw radii for the two, and probably also a bit below the full radius. Consider maybe radius * 0.9 and 0.95.